src/pages/ResetPassword.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 
-import Navbar from '../components/presentationals/Navbar.jsx';
-import AuthenticationIllustration from '../components/presentationals/AuthenticationIllustration.jsx';
-import ResetPasswordCard from '../components/presentationals/ResetPasswordCard.jsx';
+import Navbar from '../components/presentationals/Navbar/Navbar';
+import AuthenticationIllustration from '../components/presentationals/AuthenticationIllustration/AuthenticationIllustration';
+import ResetPasswordCard from '../components/presentationals/ResetPassword/ResetPasswordCard';
 import './Authentication.scss';
 
-const ResetPassword = () => {
+const ResetPassword = defaultAvatarUrl => {
```

Comment:
I think this component and `AuthenticationLayou` looks similar.

Reply:
This has the same reason above on forgot password